docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,170 @@
+-   Pulse associates all Bitbucket teams of the author of a merged pull request with the corresponding deployment, excluding teams with less than two members. Pulse only takes changes to Bitbucket teams into account on pull requests merged after those changes.
+
+## Collected data
+
+The table below lists the data that the Bitbucket integration collects from your Bitbucket workspace, together with:
+
+-   The mapping between the data collected from Bitbucket and the [Pulse data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event)
+-   The metrics that Pulse calculates from the data to display on the dashboards
+
+<table>
+<thead>
+<tr>
+<th><strong>Data collected from Bitbucket</strong></th>
+<th><strong>Mapping to Pulse data model</strong></th>
+<th><strong>Used in</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+    <td>Pull request commits</td>
+    <td>
+        <p>Changes:</p>
+        <ul>
+            <li><code>change_id</code>: commit UUID</li>
+            <li><code>time_created</code>: commit author date<sup><a href="#commit-author-date">1</a></sup></li>
+            <li><code>system</code>: repository name</li>
+        </ul>
+    </td>
+    <td>Lead time for changes on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
+</tr>
+<tr>
+    <td>Pull requests</td>
+    <td>
+        <p>Deployments:</p>
+        <ul>
+            <li><code>deploy_id</code>: pull request ID</li>
+            <li><code>system</code>: repository name</li>
+        </ul>
+    </td>
+    <td>Deployment frequency and Change failure rate on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
+</tr>
+<tr>
+    <td>Teams</td>
+    <td>
+        <p>Deployments:</p>
+        <ul>
+            <li><code>teams</code>: Bitbucket teams of the author of a deployment<sup><a href="#deployment-teams">2</a></sup></li>
+        </ul>
+    </td>
+    <td>Filters the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a> by the contributions made by the team</td>
+</tr>
+</table>
+
+<sup><span id="commit-author-date">1</span></sup>: Pulse uses the commit author's date since it is more accurate. The committer date can be changed (e.g.: fast-forward) and stop reflecting the real creation date of the change.
+
+<sup><span id="deployment-teams">2</span></sup>: Adding a new team or changing the composition of a team on Bitbucket only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted teams on Bitbucket will continue to show in Pulse.
+
+## Which permissions does Pulse need from Bitbucket? {: id="bb-permissions"}
```

Comment:
@afsmeira I will complete this section with the information you'll provide in the scope of [PUL-1507](https://codacy.atlassian.net/browse/PUL-1507).
If you prefer to provide that information here instead of creating a new internal document, please go ahead. What's easier for you.

Reply:
Check my comment above, I think it's all there.

Reply:
Thank you so much for all the details, @afsmeira . 🙏  I updated the permissions table with the provided information.